docs/extensions/extension_example.md

```diff
@@ -192,43 +190,48 @@ For our example, the output looks like this:
 accepts: [ "1.3.8", "1.0.0" ]
 ```
 
-And that's all the code we need to check for duplicates. Now we only need to
-install it and we can use it for any lockfile we encounter in the future:
+And that's all the code we need to check for duplicates. Now we can use the
+`phylum extension run` subcommand to test the extension without installing it:
 
 ```sh
-phylum extension install ./duplicates
-phylum duplicates ./package-lock.json
+phylum extension run ./duplicates ./package-lock.json
 ```
 
-Currently none of our code does any external I/O, we're completely contained
-within the Deno sandbox. Let's change that. Instead of the `console.log`, we'll
-now write our output to a file instead:
+This should then print the following error:
 
-```ts
-await Deno.writeTextFile(
-    "./duplicates.txt",
-    `${dep}: ${distinctVersions}\n`,
-    { append: true }
-);
+```text
+Extension error: Uncaught (in promise) Error: Requires read access to "./package-lock.json"
+    at async Function.parseLockfile (deno:phylum:201:16)
+    at async file:///tmp/duplicates/main.ts:12:14
 ```
 
-When replacing the `console.log` with this function call and executing our
-extension, you'll run into the following error:
+Phylum's extensions are executed in a sandbox with restricted access to
+operating system APIs. Since we want to read the lockfile from
+`./package-lock.json` with the `parseLockfile` method, we need to request read
+access to this file ahead of time. All available permissions are documented in
+the [extension manifest documentation].
 
-```text
-❗ Error: Execution failed caused by: Error: Requires write access to "./duplicates.txt"
-```
+[extension manifest documentation]: https://docs.phylum.io/docs/extension_manifest#permissions
 
-This is exactly what should have happened, since Deno's sandbox doesn't allow us
-to interact with the outside world unless we've been granted permission to do
-so. To request permissions, you'll have to edit the `PhylumExt.toml` manifest
-and add the following:
+While it would be possible to request read access to just `./package-lock.json`,
+this would only work for `package-lock.json` files defeating the purpose of
+passing the lockfile as a paramater. Instead, we request read access to all
```

Comment:
```suggestion
this would only work for `package-lock.json` files, defeating the purpose of
passing the lockfile as a parameter. Instead, we request read access to all
```

Reply:
Good catch, thanks.